docs/reference/api/mainwindow.rst

```diff
@@ -55,10 +55,20 @@ Usage
 -----
 
 A :class:`toga.MainWindow` is a :class:`toga.Window` that can serve as the main
-interface to an application. If the platform places menus inside windows, a
-:class:`toga.MainWindow` instance will display a menu bar that contains the app
-control commands (such as About, Quit, and anything else required by the
-platform's HIG). It may also contain a toolbar.
+interface to an application. A :class:`toga.MainWindow` may optionally have a toolbar.
+The presentation of :class:`toga.MainWindow` is platform dependent:
+
+* On desktop platforms that place menus inside windows (e.g., Windows, and most Linux
+  window managers), a :class:`toga.MainWindow` instance will display a menu bar that
+  contains the app control commands (such as About, Quit, and anything else required by
+  the platform's HIG).
```

Comment:
This makes user-defined commands seem like an afterthought. In fact, I don't think it's helpful to distinguish between HIG commands and user-defined commands here – that can be done on the App or Commands pages.

```suggestion
* On desktop platforms that place menus inside windows (e.g., Windows, and most Linux
  window managers), a :class:`toga.MainWindow` instance will display a menu bar that
  contains :attr:`App.commands`.
```